### PR TITLE
Accordion DAP fixes

### PR DESCRIPTION
**Pull request by aradhikaIbm**
Closes IBM/carbon-components-angular #106

Fix DAP issues for Accordion
Set the role to heading and use aria-level per guidelines.
Defaulting level to 3 since must pages have heading level 1 already.
User can override.

#### Changelog

**Changed**

* Set the role to heading and use aria-level per guidelines.

### CODE REVIEW

**Comment by aradhikaIbm** [comment]
Does stacking the @HostBindings for the display and role not work?  I was trying not impact the css the widget was using. I believe heading is not a valid value for the display css. 

Level 3 was something on I saw the example that they provided in the design. I think it also represents the heading within the page.  So most pages use 1 as the page header.

File: src/accordion/accordion-item.component.ts [hunk 1]
```diff
@@ -36,7 +36,9 @@ export class AccordionItem {
 
 	@HostBinding("class.bx--accordion__item") itemClass = true;
 	@HostBinding("class.bx--accordion__item--active") @Input() expanded = false;
-	@HostBinding("style.display") @HostBinding("attr.role") itemType = "list-item";
+	@HostBinding("style.display") itemType = "list-item";
+	@HostBinding("attr.role") roleType = "heading";
```
Comment (zvonimirfras) [inline src/accordion/accordion-item.component.ts]: `itemType` makes sense as a variable name there, `roleType` should be `role`

Comment (aradhikaIbm) [reply src/accordion/accordion-item.component.ts]: I'll make that change shortly

**Comment by zvonimirfras** [comment]
It looks like you've overridden all the changes of the previous merge with yours. Please undo that.

Your diff also grew from 3 lines to whole files... Not sure what happened there :\

**Comment by aradhikaIbm** [comment]
Will redo with a new branch. Its not clear to me the state of my branch.

**Pull request closed without merging**